doc: relocate FOTA build file docs (#6332)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/nrf_cloud.rst

Firmware over-the-air (FOTA) updates
************************************
The nRF Cloud library supports FOTA updates for your nRF9160-based device.
The :kconfig:`CONFIG_NRF_CLOUD_FOTA` option is enabled by default when :kconfig:`CONFIG_NRF_CLOUD_MQTT` is set.
This enables FOTA functionality in the application.

> In order to do its job, nRF Cloud FOTA enables additional features and libraries:

Reviewer: ```suggestion
nRF Cloud FOTA enables the following additional features and libraries:
```

Building FOTA images
====================
The build system will create the files :file:`dfu_application.zip` and/or :file:`dfu_mcuboot.zip` for a properly configured application.

> See :ref:`app_build_system` for more information about FOTA zip files.

Reviewer: Shouldn't the link target to app_build_fota instead?

Author: yes
